pkg/reconciler/common/targetnamespace.go

```diff
@@ -0,0 +1,61 @@
+package common
```

Comment:
@PuneetPunamiya I think u forgot to add License :grin:

Reply:
updated!